clang/lib/Driver/ToolChains/SYCL.cpp

```diff
@@ -307,7 +307,12 @@ void SYCL::fpga::BackendCompiler::ConstructJob(Compilation &C,
     std::string Filename(II.getFilename());
     if (II.getType() == types::TY_Tempfilelist)
       ForeachInputs.push_back(II);
-    CmdArgs.push_back(C.getArgs().MakeArgString(Filename));
+    // Add any FPGA library lists.  These come in as special tempfile lists.
```

Comment:
Suggest moving the comment to the next line so that it would tie in with the related code better

Reply:
Done.